scripts/check_module_coverage_floor.py

```
@@ -353,13 +359,18 @@ def update_baseline(
-            json.dumps(baseline, indent=2, sort_keys=False) + "\n",
-            encoding="utf-8",
-        )
+        with os.fdopen(tmp_fd, "w", encoding="utf-8") as fh:
+            fh.write(payload)
+        os.replace(tmp_name, baseline_path)
     except OSError as exc:
         print(f"ERROR: failed to write baseline file {baseline_path}: {exc}")
+        try:
+            os.unlink(tmp_name)
+        except OSError:
+            pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)